src/Builder/Visitor/PluginPositionResolver/PluginPositionResolver.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This is what I told today - according to PSR12 strict_types declaration should go after file documentation:

https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements

But future code fixer can fix such case automatically

Reply:
++